sql/core/src/test/scala/org/apache/spark/sql/ExplainSuite.scala

```diff
@@ -277,6 +277,28 @@ class ExplainSuite extends ExplainSuiteHelper with DisableAdaptiveExecutionSuite
     }
   }
 
+  test("explain formatted - check presence of subquery in case of AQE") {
```

Comment:
Could you add `SPARK-33850: ` prefix here, please?

Reply:
Ah, I'll add.